### ISSUE TITLE
Users need to use GOV.UK Verify but did not understand what GOV.UK Verify is or what it means to sign up with it

### ISSUE THREAD

**Issue by irinapencheva** [alphagov/verify-local-patterns#413]
Quote “why am I going to the GOV.UK website, I wanted to apply with Buckinghamshire”
Quote “no idea what these companies are”
Quote “I am not happy about it going to another place”
Quote “why is the Council asking someone else to verify me?”
Quote “of course I am over 20 it is an old person badge”

**Comment by colin2678** [comment]
This is linked to issue #400. Users are unsure of what Verify does.

**Comment by irinapencheva** [comment]
Thanks @colin2678 - we'll keep an issue in the CT section as well :)

**Comment by irinapencheva** [comment]
#426

**Comment by irinapencheva** [comment]
#465

**Comment by irinapencheva** [comment]
Also linked to #462

**Comment by lizziebruce** [comment]
@irinapencheva Screenshot I've just added to #461 may also resolve almost all of the user quote issues listed at top... it's already been changed in the prototype. The only one it wouldn't is “of course I am over 20 it is an old person badge” which relates to Verify hub pages. I'm not directly involved with that but am feeding local issues with Verify hub through to @anthonylord regularly. The developer I met yesterday in the Privacy training, Kevin, said the hub content is being contextualised for user relevancy but I'm not sure if that would be to the degree of adjusting security questions dependent on local service entry point or not. Definitely worth considering from a local services perspective.

**Comment by irinapencheva** [comment]
@lizziebruce - yep, sounds great. Thanks for feeding into main Verify issues as well.

On this specific issue there is probably something about how the start page works as well and what's presented there about Verify  - at the moment almost nothing. This is linked to #303

**Comment by lizziebruce** [comment]
@irinapencheva I will also add a line for the Start page to flag up that the Verify process could be a part of the application and identification docs may be needed... important to attempt to keep [Verify] separate from [Application] in people's minds though, in order to address the 'why do I need all this for a bus pass?' issue which came up a lot in UR. In actual fact they don't acutally need to answer the security questions, but they have chosen to in order not to scan in documents. I think if we can make that choice more apparent people will be less frustrated/baffled, as they made the choice to go through Verify.

**Comment by irinapencheva** [comment]
@lizziebruce - yep, fitting in with clearly creating that mental picture in people's mind of what will happen! And on service start page might be worth saying what GOV.UK Verify is. I've seen it other services as 'the new way to prove who you are online' or sth along those lines. Ant will have a good insight on that

**Comment by lizziebruce** [comment]
@irinapencheva "prove who you are online" is good. I'm not sure we can say "the new way" anymore. Ant described Verify to me as 'a kind of online passport' which really clicked for me but unfortunately is too metaphorical to use in service copy.

**Comment by irinapencheva** [comment]
@lizziebruce - agreed.. not new anymore, so happy with any wording you'd think would work

**Comment by lizziebruce** [comment]
@irinapencheva 
Content redesign for Start page. Complements content redesign of pre-Verify page to embed the idea of needing to choose either Verify or docs scan from the very first page. Also flags need to upload a photo. This content has been moved above the Apply button to increase the liklihood of it getting noticed. 

> <img width="405" alt="screen shot 2017-04-28 at 13 48 58" src="https://cloud.githubusercontent.com/assets/27814324/25529466/1375b27c-2c1a-11e7-888b-a76b9c4fd0f5.png">

**Comment by irinapencheva** [comment]
@lizziebruce - on the 1st point of 'About your application' do you think the issue about eligibility should also be mentioned? 

Something along the lines of: "Identify yourself and prove your eligibility using ..."

**Comment by lizziebruce** [comment]
@irinapencheva is there a user need for adding "prove your eligibility"? Has anyone been asking why they have to identify themselves? If so "identify yourself and prove your eligibility" sounds good.

**Comment by irinapencheva** [comment]
@lizziebruce - overall people don't seem to understand why they have to identify themselves and feel they're entitled to the pass (as seen in last round of UR), so thinking it might be worth testing overall comprehension of why they're going through Verify if the connection between proving identity and eligibility is more explicit

**Comment by lizziebruce** [comment]
@irinapencheva sure, that makes sense. I edit the copy to reflect that. Also needs further adjusting as it is the first mention of Verify and there's no explanation of what Verify is. It may work better to just say answer security questions and not mention Verify here. (I showed people on the GDS design training this page and got some feedback re no explanation of Verify here too and it came up in UR I believe.)

**Comment by suryanaga** [comment]
I think taking the mention of Verify out of the start page could be a good idea because we don't have the space to properly explain it there. 

Taking this back to the original issue of making sure users understand why Verify is being used, I agree with Lizzie that the changes made in #395 since this issue was created should hopefully solve the problems reported, so let's move this into "Recent changes have been made" until we see this confusion about Verify come up again.

**Comment by petegale** [comment]
@lizziebruce @sanjaypoyzer Just wanted to highlight a couple of things I've seen in the past that might be relevant.
1. In the past we've tried to avoid saying Verify is to let users prove who they are, as the inferred assumption is then that we don't actually trust our users. Instead we've tried to say things like 'Verify stops someone pretending to be you'
2. The other thing was about using the phrase 'the new way to...' When I've seen this tested before, it's gone some way to addressing people's concerns about Verify working in unexpected ways. It's new, so it's ok for it to be a bit unconventional

**Comment by irinapencheva** [comment]
@lizziebruce @sanjaypoyzer @petegale - could we take 5 mins tomorrow morning before the 9.30 standup to discuss this issue as it will be easier

**Comment by lizziebruce** [comment]
@irinapencheva discussing sounds good!

**Comment by lizziebruce** [comment]
@irinapencheva @petegale 

Sorry I thought I'd progressed this further. Suggest the "About" copy is refined further to:

> ## About your application
> 
> 1. Apply by answering security questions or scanning documents.
> 2. Upload a digital photo – we'll give you guidance on doing this.
> 3. Submit your application online.
> 4. We'll send your bus pass by post.

We spoke about perhaps moving this onto its own page to help users focus on reading it aka not skip it. What do you think @sanjaypoyzer @petegale?

**Comment by TracyEaton** [comment]
@lizziebruce Hi Lizzie, just a thought, but could we try removing the word digital and just say upload a photo. The word digital seemed to cause confusion?

**Comment by suryanaga** [comment]
@TracyEaton Hi Tracy, that issue is more related to #301 - I'll respond to your question there.

**Comment by lizziebruce** [comment]
@TracyEaton Hi Tracy, thanks. I think later in the journey users asked why they weren't told up front about what they would need, and so if we just put "photo" they might think they're all set when they're not. 

The photo upload users-felt-discouraged-from-start has its own issue #458 and the guidance help flow is in 301, as Sanjay's mentioned too.

**Comment by lizziebruce** [comment]
These changes to the prototype (screenshot below) should help the original issue "Users need to use GOV.UK Verify but did not understand what GOV.UK Verify is or what it means to sign up with it", but some of the UR comments relate directly to the Verify hub pages so have added that label too.

Before
![screen shot 2017-05-12 at 14 39 04](https://cloud.githubusercontent.com/assets/27814324/26000419/cd3b021e-3720-11e7-8310-758dc7b15fe7.png)

After
![screen shot 2017-05-12 at 14 33 56](https://cloud.githubusercontent.com/assets/27814324/26000282/3f7fa92a-3720-11e7-98a7-9b1a2054bd1f.png)

**Comment by colin2678** [comment]
Are we able to localise these screens? Our existing CT process does not usually require a utility bill,passport or driving licence and takes considerably less time than 20 minutes.

**Comment by lizziebruce** [comment]
@colin2678 could you tell us what documentation do you do require and how long your CT process is?

**Comment by colin2678** [comment]
We require a name, address, dob and a NI number to make an experian check. along with a photo upload or to take a photo using their webcam. This process usually takes around 4 minutes.

**Comment by suryanaga** [comment]
@colin2678 Hi Colin, just to clarify, are you talking about the alternative to Verify, ie the "Scan and upload documents" route?

**Comment by colin2678** [comment]
If the check fails then customers must send in proof of identity document (a passport, driving license, birth certificate in the same name as your application) and a proof of address  (a utility bill, bank statement, or any other official letter).

**Comment by colin2678** [comment]
@sanjaypoyzer Yes this is our existing product: https://concessioncard.net/Register_input.action It has it's flaws, but speed is not one of them.

**Comment by lizziebruce** [comment]
@colin2678 Hi Colin, I think yes that would need to be a localisation as most councils require document upload which can take users longer. We have put "up to" in front of 20 minutes to account for speedier / more digitally savvy users.

**Comment by petegale** [comment]
![screen shot 2017-05-22 at 10 34 46](https://cloud.githubusercontent.com/assets/2970019/26301846/4f5926e8-3eda-11e7-9a1e-c676e5d61acc.png)

**Comment by irinapencheva** [comment]
@petegale - Thanks for closing this issue. I will reopen however #422 as the evidence for this being addressed is still inconclusive

**Comment by colin2678** [comment]
@petegale As discussed on the call I recognise that there was better take-up of the Verify option, but did not see evidence that most CT users understood what Verify would do. To me this is a fundamental issue that creates confusion in CT users later on in the Verify process. I do not believe we can close this issue on the back of on 2 UR sessions.

**Comment by irinapencheva** [comment]
@colin2678 - We are happy to reopen this and test further at the next session

**Comment by lizziebruce** [comment]
See front-loading of explanation in #588 #602
